ci/github/action-build-and-push/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM jpetazzo/dind
+
+RUN apt-get update && apt-get install -y make curl git jq
```

Comment:
MINOR: list every lib in a different line (see [best practices in Docker](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments)), i.e

```Dockerfile
RUN apt-get update && apt-get install -y \
         make \
         curl \
         git \
         jq  
```